frontend/src/container/Version/index.tsx

```diff
@@ -60,9 +56,8 @@ function Version(): JSX.Element {
 						placeholder={t('latest_version')}
 					/>
 					<Button
-						onClick={(): void =>
-							onClickUpgradeHandler('https://github.com/SigNoz/signoz/releases')
-						}
+						href="https://github.com/SigNoz/signoz/releases"
```

Comment:
Unrelated question: 

Should we add tag link directly here ? As of now this takes to releases page but with tag we can directly see release notes of latest version

Reply:
yes we need to redirect to latest release